### PR TITLE
Quickstart guide examples 

### PR DESCRIPTION
**Pull request by suprjinx**
fixes #92

### CODE REVIEW

File: src/src/pages/Dashboard/components/AimIntegrations/AimIntegrations.tsx [hunk 1]
```diff
@@ -28,38 +28,23 @@ function AimIntegrations() {
 
   const integrations = [
     {
-      title: 'Integrate PyTorch Lightning',
```
Comment (jgiannuzzi) [inline src/src/pages/Dashboard/components/AimIntegrations/AimIntegrations.tsx:31]: Please don't remove PyTorch Lightning! This is the most popular ML framework!
You could instead add a (regular) PyTorch section.

File: src/src/pages/Dashboard/components/AimIntegrations/AimIntegrations.tsx [hunk 2]
```diff
@@ -130,9 +115,8 @@ with mlflow.start_run() as run:
     },
     {
       title: 'Integrate LightGBM',
-      docsLink: DOCUMENTATIONS.INTEGRATIONS.LIGHT_GBM,
       code: `import lightgbm as lgb
-import mlflow
+import mlflow.lightgbm
```
Comment (jgiannuzzi) [inline src/src/pages/Dashboard/components/AimIntegrations/AimIntegrations.tsx:119]: `import mlflow` is the correct import

File: src/src/pages/Dashboard/components/AimIntegrations/AimIntegrations.tsx [hunk 3]
```diff
@@ -75,43 +60,43 @@ keras_model.compile(optimizer="rmsprop", loss="mse", metrics=["accuracy"])
...
+dtest = xgb.DMatrix(X_test, label=y_test)
+
+with mlflow.start_run():
+    # train model
+    params = {
+        "objective": "multi:softprob",
+        "num_class": 3,
+        # ...
+    }
+    model = xgb.train(params, dtrain, evals=[(dtrain, "train")])
```
Comment (jgiannuzzi) [inline src/src/pages/Dashboard/components/AimIntegrations/AimIntegrations.tsx:95]: Please make this sample similar to the other ones by focusing only on `import`, `set_tracking_uri`, `autolog`, `start_run`, and `train`. Most other stuff should be replaced by ellipses. Users interested in XGBoost integration already know how to use XGBoost — they wanna know how to log their runs to FML.

**Review by jgiannuzzi — CHANGES_REQUESTED** [review]

File: src/src/pages/Dashboard/components/AimIntegrations/AimIntegrations.tsx (hunk 1 above)
Comment (suprjinx) [reply src/src/pages/Dashboard/components/AimIntegrations/AimIntegrations.tsx:31]: done -- the mlflow autologging docs refer to "Pytorch" but the example they give is Pytorch Lightening, so not adding a regular PyTorch section.

File: src/src/pages/Dashboard/components/AimIntegrations/AimIntegrations.tsx (hunk 3 above)
Comment (suprjinx) [reply src/src/pages/Dashboard/components/AimIntegrations/AimIntegrations.tsx:95]: done

File: src/src/pages/Dashboard/components/AimIntegrations/AimIntegrations.tsx [hunk 4]
```diff
@@ -29,155 +29,104 @@ function AimIntegrations() {
...
+learn = Learner(get_data_loaders(), Model(), loss_func=nn.MSELoss(), splitter=splitter)
 
 # Start MLflow session
-with mlflow.start_run() as run:
-    model.fit(epochs, learning_rate)
-
-# ...`,
+with mlflow.start_run():
+    # Train and fit with default or supplied command line arguments
+    learn.fit_one_cycle(args.epochs, args.lr)
```
Comment (jgiannuzzi) [inline src/src/pages/Dashboard/components/AimIntegrations/AimIntegrations.tsx:100]:
```suggestion
    # ...
    learn.fit_one_cycle(epochs, learning_rate)
```

File: src/src/pages/Dashboard/components/AimIntegrations/AimIntegrations.tsx [hunk 5]
```diff
@@ -29,155 +29,104 @@ function AimIntegrations() {
...
-    trainer,
-    event_name=Events.ITERATION_COMPLETED,
-    tag="train",
-    output_transform=lambda loss: {'loss': loss}
-)
-# ...`,
+
+# Start MLflow session
+with mlflow.start_run():
+    # train model
```
Comment (jgiannuzzi) [inline src/src/pages/Dashboard/components/AimIntegrations/AimIntegrations.tsx:118]:
```suggestion
    # ...
```

**Review by jgiannuzzi — APPROVED** [review]

**Pull request merged**